styles.css

```diff
@@ -72,9 +78,13 @@
 
 #services-grid {
     grid-area: services;
-    font-size: 3rem;
     color: var(--secondary-color);
     border: var(--secondary-color) solid 20px;
+    text-orientation: mixed;
```

Comment:
I would say that this CSS property is a very valid option in changing the direction of the text, makes the text pleasing toward the eye once viewed 🙌